src/main/cli/cli.c

```diff
@@ -3370,6 +3370,9 @@ void cliRxBind(char *cmdline){
         case RX_SPI_FRSKY_X:
         case RX_SPI_FRSKY_X_LBT:
 #endif
+#if defined(USE_RX_REDPINE_SPI)
```

Comment:
move this change here to fix the conflict:
https://github.com/betaflight/betaflight/blob/master/src/main/rx/rx_bind.c#L62

Reply:
Got it fixed and rebased to master to master!